### PR TITLE
06 get comments

### PR DESCRIPTION
**Pull request by OniUnderscore**

### CODE REVIEW

File: __tests__/comment.test.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          const { comments } = body;
+          expect(comments).toBeSorted({
+            descending: true,
+            key: "created_at",
+            coerce: true,
+          });
+        });
+    });
+  });
+
```
Comment (nc-jodie-cheung) [inline __tests__/comment.test.js:80]: You could add an additional test to check that it responds with an empty array when given a valid review that has no comments 👍

File: __tests__/comment.test.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments.length).toEqual(11);
+          comments.forEach((comment) => {
+            expect(comment.constructor).toEqual(Object);
+          });
+        });
+    });
+    test("Comment Objects should be returned in the expected format", () => {
```
Comment (nc-jodie-cheung) [inline __tests__/comment.test.js:29]: You could condense the two tests and add in line 23 before your forEach on line 35 👍

File: __tests__/comment.test.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        });
+    });
+    test("Comment Objects should be sorted in date descending order", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeSorted({
+            descending: true,
```
Comment (nc-jodie-cheung) [inline __tests__/comment.test.js:73]: Nice implementation of jest-sorted library here 😄

**Review by nc-jodie-cheung — COMMENTED** [review]
Nice work on this ticket, I've left some comments for you to take a look at and revise 😄
